osmaxx/conversion/converters/converter.py

```diff
@@ -2,7 +2,7 @@
 from osmaxx.conversion.converters import converter_gis
 from osmaxx.conversion.converters import converter_pbf
 from osmaxx.conversion.job_dispatcher.rq_dispatcher import rq_enqueue_with_settings
-from osmaxx.conversion_api.formats import FGDB, SHAPEFILE, GPKG, SPATIALITE, GARMIN, PBF
+from osmaxx.conversion.constants.formats import FGDB, SHAPEFILE, GPKG, SPATIALITE, GARMIN, PBF
```

Comment:
Singular (`format.FGDB`) would make sense then, **but** there's already a [Python builtin called `format()`](https://docs.python.org/3/library/functions.html#format) that I don't want to shadow.

Any suggestions for alternative names?